### ISSUE TITLE
Point out CLI included in InfluxDB 2.0 download

### ISSUE THREAD

**Issue by kelseiv** [influxdata/docs-v2#2557]
Clarify in install docs that CLI is included as part of the InfluxDB 2.0 download.

Where this update is made will be affected by this PR: https://github.com/influxdata/docs-v2/issues/2124

**Comment by Anaisdg** [comment]
Also include links to download just the CLI for Cloud users.

**Comment by sanderson** [comment]
@Anaisdg we just restructured the Cloud getting started docs a little. In the old one, it walked through signing up AND setting up the CLI / other things you want to do after you're signed up. With this change, we separated the getting started from the sign up content and the sign up process is now in its own doc. The CLI download was left in with the sign up process [here](https://docs.influxdata.com/influxdb/cloud/sign-up/#optional-download-install-and-use-the-influx-cli), but I think it would fit better in [Get started with InfluxDB cloud](https://docs.influxdata.com/influxdb/cloud/get-started/).

We could also add download links to the influx CLI landing page. The CLI's for OSS and Cloud are the same, so we'd only have to do one link. We would just need to remember to update the versions when when we cut an OSS release. OR I could add a shortcode that pulls the version from a central location so we only have to update it one place 🤔 .
